Resources/FAQ/Jobs-Discord-Asked-Questions.md

```diff
@@ -89,18 +119,20 @@ This is a loaded question, because it can mean a lot of things. Another plugin c
 
 ---
 
-### What are the placeholders for Jobs plugin?
+### Where can I find a list of all Jobs Reborn placeholders?
+
+Type: `jobs placeholders` to get the complete list in the console.
 
-In the console, type: `jobs placeholders` to get the full list.
+You can also do this in-game with the command `/jobs placeholders`
 
 ---
 
 ### I asked a question, and I am getting ignored?
 
-To get the best help you can get we need information, then Zrips on Github, or the community members on Discord can help you better.
+To get the best help you can get, we need information, then Zrips on Github or the community members on Discord can help you better.
 
 - Have some patience. Post the question, provide info, and just wait.
-- We always advise to stay current and make frequent backups!
+- We always advise you to stay current and make frequent backups!
```

Comment:
Probably not needed, as this advice is given to everyone, not specifically the reader